modules/installation-osp-verifying-external-network.adoc

```diff
@@ -0,0 +1,36 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+//
+// DNS resolution KI
+
+[id="installation-osp-verifying-external-network_{context}"]
+= Verifying public network access
```

Comment:
@iamemilio @tomassedovic Say I don't do this. What happens, and would it be obvious that it's happening if I haven't completed this procedure?

Reply:
If you don't provide any external network value during the installation, the installer will point out that it's needed.

If you provide a network name that's not of the `External` Router Type, Terraform would fail with a possibly hard-to-understand error. I haven't verified, but Terraform's errors generally aren't that great.

If you don't follow the steps in the prerequisites *and* the servers indeed cannot perform DNS lookups, the `bootkube` service in the bootstrap node will fail. This should be relatively easy to understand and figure out when you look at the must-gather logs the installer provides. However, the bootstrapping time out is set to 30 minutes, so it will be a poor first-time experience.

If you have a CIDR overlap, you might run into hard-to-understand errors during the machine boot (that you can only see with `openstack console log show <cluster-id>-bootstrap` or possibly even something way further down the line (e.g. etcd issues, some pods not coming up or behaving incorrectly).